v-next/hardhat/src/internal/builtin-plugins/solidity-test/config.ts

```diff
@@ -121,6 +123,16 @@ export async function resolveSolidityTestUserConfig(
   testsPath = typeof testsPath === "object" ? testsPath.solidity : testsPath;
   testsPath ??= "test";
 
+  const defaultRpcCachePath = path.join(
+    resolvedConfig.paths.cache,
+    "edr-cache",
```

Comment:
Should we just name it `edr` since it's going to be in a directory named `cache`? This is a non-blocking suggestion, feel free to take it or leave it.

```suggestion
    "edr",
```